subsys/suit/mci/src/suit_mci_nrf9280.c

```diff
@@ -0,0 +1,716 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <suit_mci.h>
+#include <drivers/nrfx_common.h>
+#include <suit_storage.h>
+#include <suit_storage_mpi.h>
+#include <suit_execution_mode.h>
+#ifdef CONFIG_SDFW_LCS
+#include <sdfw/lcs.h>
+#endif /* CONFIG_SDFW_LCS */
+#include <zephyr/logging/log.h>
+#include <sdfw/arbiter.h>
+
+#define MANIFEST_PUBKEY_NRF_TOP_GEN0	 0x4000BB00
+#define MANIFEST_PUBKEY_SYSCTRL_GEN0	 0x40082100
+#define MANIFEST_PUBKEY_OEM_ROOT_GEN0	 0x4000AA00
+#define MANIFEST_PUBKEY_APPLICATION_GEN0 0x40022100
+#define MANIFEST_PUBKEY_RADIO_GEN0	 0x40032100
+#define MANIFEST_PUBKEY_GEN_RANGE	 2
```

Comment:
There is no sign of `SUIT_MANIFEST_NORDIC_CELLFW` in this file, but I suspect this is intentional to bring-up the SUIT without modem manifests at this stage.

Reply:
Yes, we are still trying to get to the level of Lilium. We would first need to be at that level to enable testing of Halti with SUIT in CI.

Then we need later to extend SUIT for Halti to cover Cellular FW as well.